ecommerce_integrations/shopify/doctype/shopify_setting/shopify_setting.json

```diff
@@ -37,6 +37,7 @@
   "section_break_22",
   "html_16",
   "taxes",
+  "dont_recalculate_taxes",
```

Comment:
I think this can be the default behaviour. Shopify should be the source of truth for tax deducted. 

The configurability is only required if we want to merge tax lines.

Reply:
Actually it is the opposite way.... when we are not recalculating taxes, they are merged (because shopify is doing so)... you can try it out with my scenario order

Reply:
@ankush understandable? or shall I explain a little bit more?